### PR TITLE
chore(console,phrases): add mfa experience phrases

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

chore(console,phrases): add mfa experience phrases

## Testing

Test locally.

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/pages/SignInExperience/tabs/Content/components/ManageLanguage/LanguageEditor/LanguageDetails.tsx [hunk 1]
```diff
@@ -50,8 +51,14 @@ function LanguageDetails() {
   const translationEntries = useMemo(
     () =>
       Object.entries((isBuiltIn ? resource[selectedLanguage] : en).translation).filter(
-        // Filter out the demo app phrases in AC
-        ([groupKey]) => groupKey !== 'demo_app'
+        ([groupKey]) => {
```
Comment (simeng-li) [inline packages/console/src/pages/SignInExperience/tabs/Content/components/ManageLanguage/LanguageEditor/LanguageDetails.tsx:54]: Should implement a more general solution. e.g. A exclusive list.

File: packages/console/src/pages/SignInExperience/tabs/Content/components/ManageLanguage/LanguageEditor/LanguageDetails.tsx [hunk 2]
```diff
@@ -200,11 +207,27 @@ function LanguageDetails() {
...
+                if (groupKey === 'action') {
+                  return !['copy', 'verify_via_passkey', 'download'].includes(phraseKey);
+                }
+
+                if (groupKey === 'input') {
+                  return phraseKey !== 'backup_code';
+                }
+
+                return true;
+              })
```
Comment (simeng-li) [inline packages/console/src/pages/SignInExperience/tabs/Content/components/ManageLanguage/LanguageEditor/LanguageDetails.tsx:225]: ditto

File: packages/console/src/pages/SignInExperience/tabs/Content/components/ManageLanguage/LanguageEditor/LanguageDetails.tsx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/tabs/Content/components/ManageLanguage/LanguageEditor/LanguageDetails.tsx]: updated.

File: packages/console/src/pages/SignInExperience/tabs/Content/components/ManageLanguage/LanguageEditor/LanguageDetails.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/tabs/Content/components/ManageLanguage/LanguageEditor/LanguageDetails.tsx]: Updated.

File: packages/console/src/pages/SignInExperience/utils/constants.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+];
+
+export const hiddenLocalPhrases: readonly LocalePhraseKey[] = [
+  ...conditionalArray(
+    !isDevFeaturesEnabled &&
+      ([
+        'action.copy',
+        'action.verify_via_passkey',
+        'action.download',
+        'input.backup_code',
```
Comment (charIeszhao) [inline packages/console/src/pages/SignInExperience/utils/constants.ts:21]: I think we don't need to hide `action.copy` and `action.download`

Comment (charIeszhao) [reply packages/console/src/pages/SignInExperience/utils/constants.ts:21]: Place `verify_via_passkey` and `backup_code` in `mfa` group? This way we don't have to hide them seperately

**Review by charIeszhao — APPROVED** [review]

File: packages/console/src/pages/SignInExperience/utils/constants.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import { conditionalArray } from '@silverhand/essentials';
+
+import { isDevFeaturesEnabled } from '@/consts/env';
+
+export const hiddenLocalPhraseGroups: readonly LocalePhraseGroupKey[] = [
+  'demo_app',
+  ...conditionalArray(!isDevFeaturesEnabled && 'mfa'),
+];
+
+export const hiddenLocalPhrases: readonly LocalePhraseKey[] = [
```
[the remark this answers is not in the document]
Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/utils/constants.ts]: Updated.

File: packages/console/src/pages/SignInExperience/utils/constants.ts (hunk 3 above)
Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/utils/constants.ts:21]: the hidden feature is intended for convenient feature control during the development process. Therefore, we will retain the current implementation.

**Pull request merged**